demo/demo/src/readme.md

```diff
@@ -1,5 +1,28 @@
+<!-- test -->
 # Welcome to the Doxysphinx Demo C++ project
 
-This is the mainpage of the doxysphinx demo project.
+<!-- You can link to sphinx static content by specifing a relative path in the final output -->
+<!-- ![Logo](../../../../_static/doxysphinx_logo.svg) -->
 
-![Logo](../../../../_static/doxysphinx_logo.svg)
+<!-- however you can also use doxygens internal linking mechanism-->
+![Logo](@ref doxysphinx_logo.svg)
+
+### Intro
+
+This is the mainpage of the doxysphinx C++ demo sources.
+It is a markdown document rendered by doxygen (not sphinx).
+
+> If you use markdown pages in doxygen and want to reference e.g. classes you can use the special syntax
+> provided by doxygen-markdown (see <https://doxygen.nl/manual/markdown.html>), e.g.
+> [This is a link to the Car class](@ref doxysphinx::rst::Car).
+
+### Linking to sections
+
+You can also reference the doxygen sections manually like this (using final html document names):
+
+* [Namespace List](namespaces.html)
```

Comment:
@mb-emag Do we have some information in the documentation (not as am example) which answers the question
> How to make Doxysphinx sections like pages, modules, namespaces, annotated, files appear in the generated documentation?
Answer is imply to include the doxysphinx generated `index.rst` in a `toctree` 

Sorry, I was too lazy to search the documentation for this and raise a PR.

Reply:
@mb-emag Ping, incase the notification was missed and you are waiting for me :)

Reply:
@twodrops, sorry i missed that one.

Yes we have it documented here: <https://boschglobal.github.io/doxysphinx/docs/linking_to_doxygen.html#via-document-link>

Reply:
@mb-emag No problem. Do you we have specific topic which talks about making those Doxygen subchapters visible? I could not find it under the above link

Reply:
Ah ok guess it's too short then, i mean't this sentence here:
> This is typically used **to reference the root docs or some special page where you know the exact name** (Don’t use it for C++ Symbols, as there are better ways to link them

However, beside that we have no documentation how to link to the namespace pages, related pages section etc.
Perhaps we could add an info box right below the "inside a toctree directive" on that page where we name these standard doxygen sections? or an faq entry?

Reply:
I'll do it with this one :-)

Reply:
@twodrops - I added a tip section with the standard pages we have in the doxysphinx repo. I'm not sure what pages doxygen could have in an addition to that... should we add other ones too? (you can see the ones I've added here: https://github.com/boschglobal/doxysphinx/pull/45/commits/149b5e252ebff0a960832440d522ea9323472011)